Content.Server/Fluids/EntitySystems/PuddleSystem.cs

```diff
@@ -511,10 +513,13 @@ public Solution GetOverflowSolution(EntityUid uid, PuddleComponent? puddle = nul
                     $"{ToPrettyString(user.Value):user} threw {ToPrettyString(uid):entity} which splashed a solution {SolutionContainerSystem.ToPrettyString(solution):solution} onto {ToPrettyString(owner):target}");
             }
 
+            targets.Add(owner);
             _reactive.DoEntityReaction(owner, splitSolution, ReactionMethod.Touch);
             _popups.PopupEntity(Loc.GetString("spill-land-spilled-on-other", ("spillable", uid), ("target", Identity.Entity(owner, EntityManager))), owner, PopupType.SmallCaution);
         }
 
+        RaiseNetworkEvent(new ColorFlashEffectEvent(solution.GetColor(_prototypeManager), targets), Filter.Pvs(uid, entityManager: EntityManager));
```

Comment:
this should be in a function somewhere rather than just raising this event in random places.

Reply:
no u